ui/app/common/services/allergyService.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+angular.module('bahmni.common.util')
+    .factory('allergyService', ['$http', 'appService', function ($http, appService) {
```

Comment:
Why we need this file? I didn't see any use case for this and also we already have the same allergyService file under  `ui/app/clinical/common/services`. https://github.com/Bahmni/openmrs-module-bahmniapps/blob/master/ui/app/clinical/common/services/allergyService.js

Reply:
actually initially the allergy service was under ui/app/clinical/common/services. but at later point of time, we moved it to ui/app/common/services so that it can be accessed in other modules too. since we merged the master branch later to this PR. it might've got missed. I'll remove the one on this path ui/app/clinical/common/services